src/pages/TransactionReceiptPage.tsx

```diff
@@ -31,6 +31,7 @@ function TransactionReceipt({transaction, report, reportMetadata = {isLoadingIni
     const imageSource = tryResolveUrlFromApiRoot(receiptURIs.image ?? '');
 
     const isLocalFile = receiptURIs.isLocalFile;
+    const readonly = route.params.readonly ?? false;
```

Comment:
route param returns string and we are considering it as a boolean which causes a bug related to it [here](https://github.com/Expensify/App/issues/50980).